src/main/kotlin/com/cognifide/gradle/aem/deploy/responses/ListResponse.kt

```diff
@@ -1,4 +1,4 @@
-package com.cognifide.gradle.aem.deploy
+package com.cognifide.gradle.aem.deploy.responses
```

Comment:
I prefer to organize packages by domain/functionality rather than type of code / structural unit.

I mean better subpackage will be `install` with `InstallTask` and `InstallResponse`, because if you will clean the code will able to just remove whole package and second thing... code reviewer will need only to review that subpackage to find out about eveyrything related with installation feature. generally I'd like to warmly encourage you to follow that practice. It can help very much with organizing code which will grow over years...